README.md

```diff
@@ -59,6 +59,20 @@ Or override in step options
 
 Adds `single` or `multiple` to the locals to describe the number of errors for pluralisation of error messages.
 
+#### Exposes meta to templates
+
+Add a meta object to step config to expose the contents of the object to the template via locals. Useful for generating templates programmatically
+
```

Comment:
Add a `meta` object to a step to expose configurable key/value pairs in the template.